<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopPage</class>
 <widget class="QWidget" name="ShopPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>商城</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#ShopPage {
    background: qlineargradient(spread:pad,x1:1,y1:0,x2:0,y2:1,stop:0 #e8e6fb, stop:0.6 #d6c9f7, stop:1 #b8a4e6);
    font-family: "微软雅黑";
}
   </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="searchbar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QWidget#searchbar {
    background: rgba(239,235,251,0.85);
    border-radius: 16px;
    font-family: "微软雅黑";
}
      </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">
QLineEdit {
    background-color: #f5f3fa;
    border: 1px solid #d6c9f7;
    border-radius: 10px;
    color: #6c5b7b;
    font-size: 14pt;
    font-family: "微软雅黑";
    padding: 6px 10px;
}
QLineEdit:focus {
    background-color: #fff;
    border: 1.5px solid #a18cd1;
}
         </string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>搜索商品</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    border-radius: 16px;
    border: none;
    color: #fff;
    background: qlineargradient(spread:pad,x1:1,y1:0,x2:0,y2:1,stop:0 #b8a4e6,stop:1 #a18cd1);
    font: bold 14pt "微软雅黑";
    padding: 6px 18px;
    box-shadow: 0 2px 8px rgba(160,140,209,0.10);
}
QPushButton:hover {
    background: #a18cd1;
}
         </string>
        </property>
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="list_classification">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QListWidget {
    border: 1.5px solid #a18cd1;
    background: #f5f3fa;
    border-radius: 12px;
    color: #6c5b7b;
    padding: 6px 0;
    font-family: "微软雅黑";
    font-size: 15pt;
}
QListWidget::Item {
    padding: 12px 4px;
    border-radius: 8px;
    margin-bottom: 6px;
}
QListWidget::Item:hover {
    background: #e8e6fb;
    color: #a18cd1;
}
QListWidget::item:selected {
    background: qlineargradient(spread:pad,x1:1,y1:0,x2:0,y2:1,stop:0 #b8a4e6,stop:1 #a18cd1);
    color: #fff;
}
        </string>
       </property>
       <property name="resizeMode">
        <enum>QListView::Fixed</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
